### PR TITLE
Plans section

### PR DESCRIPTION
**Pull request by jamesm-macrometa**
The purpose of this PR is to explain how Playground users can upgrade their plans to Enterprise or Scale.

### CODE REVIEW

File: docs/account-management/plans/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title: Plans
+---
+
+This section explains the differences between each Macrometa plan, and how to view and change your own plan.
+
+For more information about limits, refer to [Tenant Quotas and Limits](../../references/quotas.md)
+
+- **Playground -** A test environment to familiarize yourself with Macrometa GDN. Production use is disabled.
+- **Scale -** The standard production package. Pay only for what you need, and increase or decrease your limits as you go.
+- **Enterprise -** Remove limits from your account.
```
Comment (jamesm-macrometa) [inline docs/account-management/plans/index.md:12]: @mikeridley Can you take a look at these three bullet points and let me know if this is an accurate description of the different Plans?  I'm still a little fuzzy on the exact differences between Scale and Enterprise, but the impression I get is that Scale has limits and Enterprise does not. 

Feel free to suggest alternate wording if you like.  Thanks!

**Review by diana-macrometa — APPROVED** [review]

**Comment by diana-macrometa** [comment]
@jamesm-macrometa, please update the link in the release notes under Scale Tier Self-Managed Upgrade to link to the new section.

File: docs/release-notes/release-notes-0-17-12.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ The first time you log in to Macrometa, the system now displays a welcome page w
 
 You can now upgrade and manage your scale tier payment plan in the Account page in the Macrometa web console.
 
-[Topic name](link) and [topic name](link) were updated as a result of this change.
+The [Plans](../account-management/plans/index.md) topic was created as a result of this change.
```
Comment (jamesm-macrometa) [inline docs/release-notes/release-notes-0-17-12.md:20]: @diana-macrometa How's this for the Release Notes link?

Comment (diana-macrometa) [reply docs/release-notes/release-notes-0-17-12.md:20]: Perfect.

**Pull request merged**